src/Image.cc

```diff
@@ -272,14 +272,14 @@ NAN_METHOD(Image::SetSource){
         argv[0] = Nan::Error(Nan::New(cairo_status_to_string(status)).ToLocalChecked());
       }
       Local<Context> ctx = Nan::GetCurrentContext();
-      Nan::Call(onerrorFn.As<Function>(), ctx->Global(), 1, argv).ToLocalChecked();
+      Nan::Call(onerrorFn.As<Function>(), ctx->Global(), 1, argv);
     }
   } else {
     img->loaded();
     Local<Value> onloadFn = Nan::Get(info.This(), Nan::New("onload").ToLocalChecked()).ToLocalChecked();
     if (onloadFn->IsFunction()) {
       Local<Context> ctx = Nan::GetCurrentContext();
-      Nan::Call(onloadFn.As<Function>(), ctx->Global(), 0, NULL).ToLocalChecked();
+      Nan::Call(onloadFn.As<Function>(), ctx->Global(), 0, NULL);
```

Comment:
Nice find, thanks for the fix! Do you know for sure if this is the correct change, vs using [`Nan::TryCatch`](https://github.com/nodejs/nan/blob/master/doc/errors.md#api_nan_try_catch) and re-throwing?

Reply:
> Do you know for sure if this is the correct change, vs using [`Nan::TryCatch`](https://github.com/nodejs/nan/blob/master/doc/errors.md#api_nan_try_catch) and re-throwing? (If you don't know for sure, I'll try to investigate.)

I do not know. According to my testing, the solution I implemented does propagate exceptions in JavaScript land.

Reply:
I'm trying out this PR with resemblejs and is still crashing.

Reply:
Looks like this is safe/correct.